packages/next-plugin-sentry/src/on-error-client.js

```diff
@@ -0,0 +1,11 @@
+import { withScope, captureException, captureMessage } from '@sentry/nextjs';
+
+export default async function onErrorClient({ err }) {
+  withScope(scope => {
+    if (err instanceof Error) {
+      captureException(err);
+    } else {
+      captureMessage(err.message);
```

Comment:
Hmm, why does this assume that it if it is not an `Error` then it has a message attribute?!

Couldn't this end up like `captureMessage(undefined)`?

Reply:
If `err` is not an `Error`, it's an object with `message`, `name` and `statusCode`.